### PR TITLE
Add provisional schedule for Swansea 2023 meeting

### PR DESCRIPTION
**Pull request by manics**
If everyone's happy with the rest of the timings we can fill in the `14:00 - 16:30  Parallel sessions/breakouts, including breaks` with potential topics later.

### CODE REVIEW

File: docs/events/wg_workshops/2023-09-04-september-meeting/index.md [hunk 1]
```diff
@@ -26,11 +26,36 @@ The meeting is open to everyone, and will be attended by a range of stakeholders
...
+
+```{list-table}
+:widths: 100 300
+:header-rows: 0
+
+- - 10:00 - 10:15
+  - Arrival
+- - 10:15 - 10:30
+  - Welcome and introductions
+- - 10:30 - 11:13
```
Comment (harisood) [inline docs/events/wg_workshops/2023-09-04-september-meeting/index.md:43]:
```suggestion
- - 10:30 - 11:15
```

File: docs/events/wg_workshops/2023-09-04-september-meeting/index.md [hunk 2]
```diff
@@ -26,11 +26,36 @@ The meeting is open to everyone, and will be attended by a range of stakeholders
...
+  - **Break**
+- - 11:30 - 12:45
+  - Lightning talks
+- - **12:45 - 14:00**
+  - **Lunch**
+- - 14:00 - 16:30
+  - Parallel sessions/breakouts, including breaks. <!--See &lt;todo&gt; for topics-->
+- - 16:30 - 17:00
+  - Wrap-up of parallel sessions/breakouts
+- - 17:00 - 17:30
```
Comment (harisood) [inline docs/events/wg_workshops/2023-09-04-september-meeting/index.md:55]: Do you think an hour for wrapping up breakouts and closing remarks is too long?

Comment (manics) [reply docs/events/wg_workshops/2023-09-04-september-meeting/index.md]: Changed to:
```
- - 14:00 - 16:45
  - Parallel sessions/breakouts, including breaks. <!--See &lt;todo&gt; for topics-->
- - 16:45 - 17:15
  - Conclusions, actions and next steps
- - 17:15 - 17:30
  - Closing remarks
```

What do you think?

Comment (harisood) [reply docs/events/wg_workshops/2023-09-04-september-meeting/index.md]: Yeah I back this! then we could theoretically do 3x 45 mins with 2x 15 min breaks for breakouts

**Pull request merged**